dev-qt/qtcore/qtcore-5.8.9999.ebuild

```diff
@@ -36,6 +36,7 @@ QT5_TARGET_SUBDIRS=(
 
 src_configure() {
 	local myconf=(
+		$(usex icu '-no-iconv' '-iconv')
```

Comment:
we could also just merge this with the following `qt_use icu` in a single `usex icu ...`